client/src/pages/result/Result.css

```diff
@@ -0,0 +1,24 @@
+.table{
+    width: 60%;
+    margin: 0 auto; 
+     
+}
+Button{
```

Comment:
Is this used? If not better to remove as it will make the code harder to read in the future